packages/web/src/common/store/pages/collection/sagas.js

```diff
@@ -27,7 +27,7 @@ const { fetchCollection, fetchCollectionSucceeded, fetchCollectionFailed } =
 
 function* watchFetchCollection() {
   yield takeLatest(collectionActions.FETCH_COLLECTION, function* (action) {
-    const { id: collectionId, permalink } = action
+    const { id: collectionId, permalink, fetchLineup } = action
```

Comment:
oh interesting, i see! okay hmm could we not have a useEffect that just calls fetchLineupMetada like we have with desktop? i honestly like what youve done here though which just ensures fechLineup gets called in addition to fetchCollection :)

Reply:
for some reason there was a prefix error so I did it this way. controls the flow this way which i think is what fixes the error